public/index.html

```diff
@@ -5,10 +5,18 @@
     <title>Samsung Website</title>
   </head>
   <body>
-    <h1>Samsung</h1>
-    <div id='carousel'></div>
-      <script type="text/javascript" src="http://127.0.0.1:3001/1/webpack.bundle.js"></script>
-    <div id='details'></div>
-      <script type="text/javascript" src="http://127.0.0.1:3002/1/bundle.js"></script>
-  </body>
+    <h1 style='font-size: 30px; font-family: Arial, Helvetica, sans-serif; margin-left: 5%;'>SOMESUNG</h1>
+    <div style='width: 90%; margin-left: 5%;'>
+      <div style="background-color: black; width: 100%; height: 20px; margin-bottom: 30px;"></div>
+      <div style="height: 1150px;">
+        <div id='carousel' style="float: left; width: 60%; margin-left: 30px; margin-top: 30px; position: -webkit-sticky; position: sticky;
+        top: 30px;"></div>
+          <script type="text/javascript" src="http://127.0.0.1:3001/1/webpack.bundle.js"></script>
+        <div id='detailsMain'style="float: right; width: 35%; margin-right: 30px;"></div>
+          <script type="text/javascript" src="http://127.0.0.1:3002/1/bundle.js"></script>
+      </div>
+      <div id='app' style="clear: left;"></div>
+        <script type="text/javascript" src="http://127.0.0.1:3006/1/bundle.js"></script>
+      </div>
+      </body>
```

Comment:
The indenting of the last div tag and the body tag are off and it's a bit confusing. The script tags are all also indented oddly.